DOCSUP-9041: Update the SQL-statement EXPLAIN PLAN (#24278)
ClickHouse/ClickHouse: docs/en/sql-reference/statements/explain.md

Settings:

-   `header` — Prints output header for step. Default: 0.
-   `description` — Prints step description. Default: 1.

> -   `indexes` — Shows used indexes, the number of filtered parts, and granules for every index applied. Default: 0. Supported for [MergeTree](../../engines/table-engines/mergetree-family/mergetree.md) tables.

Reviewer: Correct statement is `the number of filtered parts and the number of filtered granules`.
And I don't know if it is the same :)

!!! note "Note"
    Step and query cost estimation is not supported.

> The result JSON is a tree representing query plan. Every query plan node is a dictionary that always has the keys `Node Type` and `Plans`. `Node Type` is a string with a step name. `Plans` is an array with child step descriptions. Other optional keys may be added depending on node type and settings.
> 
> The following examples show the output of the query plan steps when `json` = 1 along with other settings.

Reviewer 2: ```suggestion
When `json` is set to 1, the query plan is represented in JSON format. Every node is a dictionary that always has the keys `Node Type` and `Plans`. `Node Type` is a string with a step name. `Plans` is an array with child step descriptions. Other optional keys may be added depending on node type and settings.
```
